cmd/installer/root_pack_add_test.go

```diff
@@ -390,30 +390,6 @@ func TestAddPack(t *testing.T) {
 		})
 	})
 
-	t.Run("test installing pack with license disagreed", func(t *testing.T) {
```

Comment:
I agree that now cpackget should **NOT** return an error when user disagrees with the license. But I still think it's a valid test to check if the pack did **NOT** get installed after user's decision.

Perhaps you could change this test, from
```golang
// Sanity check
assert.NotNil(err)
assert.Equal(errs.ErrEula, err)
assert.False(utils.FileExists(installer.Installation.PackIdx))
```

to

```golang
// Sanity check
assert.Nil(err)
assert.False(utils.FileExists(installer.Installation.PackIdx))
```

Reply:
Brought back that test and noticed the PackIdx was actually getting modified, nice catch! Used errs.ErrEula just to internally sign the install() was canceled and it's working as intended again.

Reply:
a-ha! Testing is never too much :p 

Thanks for checking in @lud0v1c !